docs/10_0_0/core/multiviewstate.md

```diff
@@ -12,6 +12,29 @@ Following components support MultiViewState:
 
 Per default MultiViewState is stored per session. As an alternative it can be stored per client window to support multi-window usage.
 
+## General Requirements
+
+MultiViewState requires a form to submit state via AJAX. When looking for a form to use, components will search in this order:
+
+1. If an explicit form ID is defined in the component configuration, it will use that form
+2. Otherwise, it will look for the closest parent form of the component
+3. If no parent form exists, it will use the first form found in the document
+
+If no form is available, MultiViewState will not function correctly and each AJAX request will throw an HTTP 405 Method Not Allowed. Therefore, ensure your components with MultiViewState enabled are placed within a form element if possible.  
+
+Alternatively, you can place an empty form at the start of your page to provide a fallback form for submission.
```

Comment:
i would not add this explicitly to those docs. better add it to the AJAX docs in general and add a link

Reply:
Oh I moved the docs to AJAX processing and added a link to MVS as its probably right anway?

OK you want me to remove the first form hack?